frame/tips/src/tests.rs

```diff
@@ -539,3 +574,16 @@ fn genesis_funding_works() {
 		assert_eq!(Treasury::pot(), initial_funding - Balances::minimum_balance());
 	});
 }
+
+#[test]
+fn tip_new_cannot_be_used_twice_second_instance() {
+	new_test_ext().execute_with(|| {
+		Balances::make_free_balance_be(&Treasury1::account_id(), 210);
+		assert_eq!(Balances::free_balance(&Treasury1::account_id()), 210);
+		assert_ok!(Tips1::tip_new(Origin::signed(10), b"awesome.dot".to_vec(), 6, 10));
+		assert_noop!(
+			Tips1::tip_new(Origin::signed(11), b"awesome.dot".to_vec(), 6, 10),
+			Error::<Test, Instance1>::AlreadyKnown
+		);
+	});
```

Comment:
This is close. You should show that the balance of `Treasury1` was affected by this tip, and the balance of `Treasury2` was not.

Reply:
The preceeding Treasury overwrites the former
```Rust
Balances::make_free_balance_be(&Treasury::account_id(), 101);
Balances::make_free_balance_be(&Treasury1::account_id(), 210)
...
assert_eq!(Balances::free_balance(Treasury::account_id()), 200);

```
This passes

Reply:
Right... but I want this in the test. The whole point of the test is to show the two treasuries are independently affected by the two bounties/tips.

Reply:
just to be clear i should include the above snippet as is in the additional test for tips & bounty?

Reply:
@shawntabrizi, resolved?